jerry-core/debugger/debugger.c

```diff
@@ -26,6 +28,15 @@
 #ifdef JERRY_DEBUGGER
 
 /**
+ * HAVE_TIME
+ */
+#ifdef HAVE_TIME_H
+#include <time.h>
+#elif defined (HAVE_UNISTD_H)
+#include <unistd.h>
+#endif /* HAVE_TIME */
```

Comment:
this comment is needed, but it should match the predicate of the `#ifdef`, i.e., `#endif /* HAVE_TIME_H */`